_includes/manuals/nightly/3.6_upgrade.md

```diff
@@ -101,6 +101,12 @@ Enable the Foreman module:
 dnf -y module enable foreman:el8
 {% endhighlight %}
 
+Switch to the PostgreSQL 13 module:
+
+{% highlight bash %}
+dnf -y module switch-to postgresql:13
```

Comment:
This needs to happen before `dnf -y module enable foreman:el8`, otherwise it fails:
```console
# dnf install https://yum.theforeman.org/releases/latest/el8/x86_64/foreman-release.rpm 
# dnf module enable foreman:el8
# dnf module install postgresql
# dnf install https://yum.theforeman.org/releases/nightly/el8/x86_64/foreman-release.rpm
# dnf module enable foreman:el8
Failed to set locale, defaulting to C.UTF-8
Foreman nightly                                                                                                                                       4.0 MB/s | 1.3 MB     00:00    
Foreman plugins nightly                                                                                                                               6.5 MB/s | 1.9 MB     00:00    
Last metadata expiration check: 0:00:01 ago on Tue Mar 12 12:35:59 2024.
Modular dependency problem:

 Problem: module foreman:el8:999920240311215200:6b5f9578.x86_64 from foreman requires module(postgresql:13), but none of the providers can be installed
  - module postgresql:12:8040020201220000948:9f9e2e7e.x86_64 from appstream conflicts with module(postgresql:13) provided by postgresql:13:8040020210222041810:9f9e2e7e.x86_64 from appstream
  - module postgresql:13:8040020210222041810:9f9e2e7e.x86_64 from appstream conflicts with module(postgresql:12) provided by postgresql:12:8040020201220000948:9f9e2e7e.x86_64 from appstream
  - module postgresql:12:8040020201220000948:9f9e2e7e.x86_64 from appstream conflicts with module(postgresql:13) provided by postgresql:13:8050020210609163519:b4937e53.x86_64 from appstream
  - module postgresql:13:8050020210609163519:b4937e53.x86_64 from appstream conflicts with module(postgresql:12) provided by postgresql:12:8040020201220000948:9f9e2e7e.x86_64 from appstream
  - module postgresql:12:8040020201220000948:9f9e2e7e.x86_64 from appstream conflicts with module(postgresql:13) provided by postgresql:13:8050020210624151740:b4937e53.x86_64 from appstream
  - module postgresql:13:8050020210624151740:b4937e53.x86_64 from appstream conflicts with module(postgresql:12) provided by postgresql:12:8040020201220000948:9f9e2e7e.x86_64 from appstream
  - module postgresql:12:8040020201220000948:9f9e2e7e.x86_64 from appstream conflicts with module(postgresql:13) provided by postgresql:13:8050020210719144526:b4937e53.x86_64 from appstream
  - module postgresql:13:8050020210719144526:b4937e53.x86_64 from appstream conflicts with module(postgresql:12) provided by postgresql:12:8040020201220000948:9f9e2e7e.x86_64 from appstream
  - module postgresql:12:8040020201220000948:9f9e2e7e.x86_64 from appstream conflicts with module(postgresql:13) provided by postgresql:13:8060020211110235159:d63f516d.x86_64 from appstream
  - module postgresql:13:8060020211110235159:d63f516d.x86_64 from appstream conflicts with module(postgresql:12) provided by postgresql:12:8040020201220000948:9f9e2e7e.x86_64 from appstream
  - module postgresql:12:8040020201220000948:9f9e2e7e.x86_64 from appstream conflicts with module(postgresql:13) provided by postgresql:13:8060020211213144351:d63f516d.x86_64 from appstream
  - module postgresql:13:8060020211213144351:d63f516d.x86_64 from appstream conflicts with module(postgresql:12) provided by postgresql:12:8040020201220000948:9f9e2e7e.x86_64 from appstream
  - module postgresql:12:8040020201220000948:9f9e2e7e.x86_64 from appstream conflicts with module(postgresql:13) provided by postgresql:13:8060020220601142612:d63f516d.x86_64 from appstream
  - module postgresql:13:8060020220601142612:d63f516d.x86_64 from appstream conflicts with module(postgresql:12) provided by postgresql:12:8040020201220000948:9f9e2e7e.x86_64 from appstream
  - conflicting requests
Error: Problems in request:
Modular dependency problems:

 Problem: module foreman:el8:999920240311215200:6b5f9578.x86_64 from foreman requires module(postgresql:13), but none of the providers can be installed
  - module postgresql:12:8040020201220000948:9f9e2e7e.x86_64 from appstream conflicts with module(postgresql:13) provided by postgresql:13:8040020210222041810:9f9e2e7e.x86_64 from appstream
  - module postgresql:13:8040020210222041810:9f9e2e7e.x86_64 from appstream conflicts with module(postgresql:12) provided by postgresql:12:8040020201220000948:9f9e2e7e.x86_64 from appstream
  - module postgresql:12:8040020201220000948:9f9e2e7e.x86_64 from appstream conflicts with module(postgresql:13) provided by postgresql:13:8050020210609163519:b4937e53.x86_64 from appstream
  - module postgresql:13:8050020210609163519:b4937e53.x86_64 from appstream conflicts with module(postgresql:12) provided by postgresql:12:8040020201220000948:9f9e2e7e.x86_64 from appstream
  - module postgresql:12:8040020201220000948:9f9e2e7e.x86_64 from appstream conflicts with module(postgresql:13) provided by postgresql:13:8050020210624151740:b4937e53.x86_64 from appstream
  - module postgresql:13:8050020210624151740:b4937e53.x86_64 from appstream conflicts with module(postgresql:12) provided by postgresql:12:8040020201220000948:9f9e2e7e.x86_64 from appstream
  - module postgresql:12:8040020201220000948:9f9e2e7e.x86_64 from appstream conflicts with module(postgresql:13) provided by postgresql:13:8050020210719144526:b4937e53.x86_64 from appstream
  - module postgresql:13:8050020210719144526:b4937e53.x86_64 from appstream conflicts with module(postgresql:12) provided by postgresql:12:8040020201220000948:9f9e2e7e.x86_64 from appstream
  - module postgresql:12:8040020201220000948:9f9e2e7e.x86_64 from appstream conflicts with module(postgresql:13) provided by postgresql:13:8060020211110235159:d63f516d.x86_64 from appstream
  - module postgresql:13:8060020211110235159:d63f516d.x86_64 from appstream conflicts with module(postgresql:12) provided by postgresql:12:8040020201220000948:9f9e2e7e.x86_64 from appstream
  - module postgresql:12:8040020201220000948:9f9e2e7e.x86_64 from appstream conflicts with module(postgresql:13) provided by postgresql:13:8060020211213144351:d63f516d.x86_64 from appstream
  - module postgresql:13:8060020211213144351:d63f516d.x86_64 from appstream conflicts with module(postgresql:12) provided by postgresql:12:8040020201220000948:9f9e2e7e.x86_64 from appstream
  - module postgresql:12:8040020201220000948:9f9e2e7e.x86_64 from appstream conflicts with module(postgresql:13) provided by postgresql:13:8060020220601142612:d63f516d.x86_64 from appstream
  - module postgresql:13:8060020220601142612:d63f516d.x86_64 from appstream conflicts with module(postgresql:12) provided by postgresql:12:8040020201220000948:9f9e2e7e.x86_64 from appstream
  - conflicting requests
```

Reply:
Thinking more about this, `dnf module enable foreman:el8` is not even needed during upgrades, as that module is already enabled, and enabling it again doesn't do anything (as we see).
Shall we drop it?

Reply:
I'd be tempted to keep it in because it yells at you if the module isn't enabled (correctly).

We should make it consistent with docs.t.o though.

Reply:
We actually have that in the docs.t.o version too, so keeping it!